util/execution/table_generator/table_generator.cpp

```diff
@@ -456,7 +457,7 @@ void TableGenerator::GenerateMiniRunnersData(const runner::MiniRunnersSettings &
             col_nums.emplace_back(dist.second);
```

Comment:
Somehow the comment is not added... I think there's this removing of the types with 0 column that is redundant right now with the new `GenerateTableName` refactor?